static/app/components/metrics/mriSelect/index.tsx

```diff
@@ -214,6 +216,14 @@ export const MRISelect = memo(function MRISelect({
     [onChange]
   );
 
+  const maxLength = useMemo(() => {
+    if (breakpoints.small) {
+      // at least small size screen, no problem with fitting 100 characters
+      return 100;
+    }
+    return 35; // xs size screen, width less than theme.breakpoints.small
```

Comment:
```suggestion
    return 22; // xs size screen, width less than theme.breakpoints.small
```

I believe that 22 would be better here